leasing/models/rent.py

```diff
@@ -15,3 +15,24 @@ class Rent(TimestampedModelMixin):
     end_date = models.DateField(verbose_name=_("Effective end date"), null=True, blank=True)
     amount = models.DecimalField(verbose_name=_("Amount"), null=True, blank=True, max_digits=12,
                                  decimal_places=2, help_text=_("Per month if rent type is Fixed or Index."))
+
+    def get_amount_for_period(self, period_start_date, period_end_date):
+        if not self.amount or self.type in (RentType.FREE, RentType.MANUAL) or \
+                (self.start_date and self.start_date > period_end_date) or \
+                (self.end_date and self.end_date < period_start_date):
```

Comment:
Use parenthesis rather than backslashes:
```python
if not self.amount or self.type in (RentType.FREE, RentType.MANUAL) or (
        (self.start_date and self.start_date > period_end_date) or
        (self.end_date and self.end_date < period_start_date)):
```

Quoting PEP-8 (https://www.python.org/dev/peps/pep-0008/):
> The preferred way of wrapping long lines is by using Python's implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation.